src/mrack/outputs/ansible_inventory.py

```diff
@@ -206,8 +207,9 @@ def create_inventory(self):
                 if not added:  # group doesn't exist
                     add_group(inventory, group, host.name)
 
-            # Main record belongs in "all" group
-            all_group["hosts"][host.name] = self.create_ansible_host(host.name)
+            # Main record belongs in "all" group but use fqdn if possible
+            host_name = meta_host["name"] if is_windows_host(meta_host) else host.name
+            all_group["hosts"][host_name] = self.create_ansible_host(host.name)
```

Comment:
As said in the previous review., we should be changing the behavior only in OpenStack vm name. Thus this should not be here.
AKA, keeping the exact value from metadata is good.

Reply:
>We are passing `host.name` as `name` parameter which is `Host` object attribute `name` and this is set based on response from server.
If we ask for name `ad.domain.com` the `host.name` will be `ad` as change is we ask for this name in OpenStack.